core/src/main/java/org/elasticsearch/search/suggest/completion/CompletionSuggestionBuilder.java

```diff
@@ -255,8 +257,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
      * Sets the prefix to provide completions for.
      * The prefix gets analyzed by the suggest analyzer.
      */
+    @Override
     public CompletionSuggestionBuilder prefix(String prefix) {
-        super.setPrefix(prefix);
+        super.prefix(prefix);
```

Comment:
@abeyad to my knowledge there was an ungoing debate. Last issue I can find is this https://github.com/elastic/elasticsearch/issues/14266 but it also states that it might be good to do this in one go.